### PR TITLE
fix: Profile view stats hover bug

### LINKED ISSUE #5023: profile views stats
### Description

The stats/graphs of profile views are not displaying properly when hovering over them.

### Screenshots

https://user-images.githubusercontent.com/100037271/220819264-f0f96649-fdbc-430d-b066-ad6efdaf0a90.mp4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by abdulsamad**
## Fixes Issue

Closes #5023 Fixed profile view stats chart not displaying on hover bug 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5092"></a>

### CODE REVIEW

**Comment by abdulsamad** [comment]
Sorry. I saw some pull requests on other issues without being assigned so I thought It was okay. Just read that we're not supposed to work on issues until assigned. Can you please assign it to me now?

**Review by iamthecloverly — APPROVED** [review]

File: pages/account/statistics.js [hunk 1]
```diff
@@ -139,8 +139,8 @@ export default function Statistics({ data, profile }) {
                 <CartesianGrid strokeDasharray="3 3" />
                 <XAxis dataKey="date" />
                 <YAxis />
-                <Bar dataKey="views" fill="#82ca9d" />
                 <Tooltip />
+                <Bar dataKey="views" fill="#82ca9d" />
```
Comment (eddiejaoude) [inline pages/account/statistics.js:143]: Oh I didn't realise the order was important, thank you 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by eddiejaoude** [comment]
> Sorry. I saw some pull requests on other issues without being assigned so I thought It was okay. Just read that we're not supposed to work on issues until assigned. Can you please assign it to me now?

sure, I can't seem to assign it to you, could you please comment in the issue and I will assign it to you

**Pull request merged**